modules/network-observability-flowcollector-about.adoc

```diff
@@ -0,0 +1,121 @@
+<4> The `spec.quickFilters` specification defines filters that show up in the web console. The `Application` filter keys,`src_namespace` and `dst_namespace`, are negated `!`, so the `Application` filter shows shows all traffic that _do not_ originate from, nor destinate to, any `openshift-` or `netobserv` namespaces.
+
+= Flow Collector Configuration Parameters
+The fields for the Flow Collector custom resource are described in the following table.
+
+.FlowCollector custom resource configuration parameters
+[cols="1,1,8a",options="header"]
+|===
+
+|Field
+|Type
+|Description
+
+|`spec.agent.type`
+|`string`
+|Specifies the agent for flows extraction. The default value is `EBPF`, as it is the only {product-title} supported agent.
+
+|`spec.agent.ebpf`
+|`integer`
+|Specifies the the settings related to the eBPF-based flow reporter. The following eBPF settings are helpful for managing compute resources when continuously sampled flow collection might consume a large amount of resources:
+
+* `ebpf.sampling` specifies the sampling rate of the flow reporter. A value of 100 means one flow every 100 is sampled. A value of 1 means all flows are sampled. The lower the value, the increase in returned flows and the accuracy of derived metrics. The default value is 50. 
+* `ebpf.resources`specifies the compute resources required by the container. The default value is 800 MB, which is sufficient for most medium-sized clusters
+* `ebpf.cacheMaxFlows` is a string that specifies the maximum duration that flows are aggregated in the eBPF agent before being forwarded to the processor. The higher the `cacheMaxFlows` value, the less traffic the eBPF agent generates, which reduces resource demands; however note that memory consumption is higher which may create latency in the flow collection.
+* `ebpf.cacheActiveTimeout` specifies the maximum number of flows that can be kept in the eBPF agent memory. If the number of flows being aggregated reaches this number, they are flushed to the Flow Processor, even if the `cacheActiveTimeout` period is not reached.The higher the `cacheActiveTimeout` value, the less traffic the eBPF agent generates, which reduces resource demands. 
+* `ebpf.interfaces` and `ebpf.excludeInterfaces` specify the interface names from where flows are collected. You can reduce the overall observed traffic by restricting or excluding interfaces, which can help with resrouce management. If empty, the agent will fetch all the interfaces in the system, excepting the ones listed in ExcludeInterfaces. If an entry is enclosed by slashes (e.g. `/br-/`), it will match as regular expression, otherwise it will be matched as a case-sensitive string.
+* `ebpf.debug.env` specifies custom environment variables to the
+Flowlogs-Pipeline pod. This field is useful for passing
+some concrete performance-tuning options, such as GOGC and
+GOMAXPROCS.
+
+
+|`spec.consolePlugin.qi`
```

Comment:
There's a mistake here, nothing named `qi`, given the description it should be `spec.loki`

Reply:
Yea, sorry about that. Its my WIP mess 🥴 . I committed as I was adding more things bc I needed to fix something that was breaking my build, but the reference table but wasn't quite complete and yeah I had interrupted another row to add more configs.